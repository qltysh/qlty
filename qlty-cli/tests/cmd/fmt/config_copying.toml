bin.name = "qlty"
args = ["fmt", "--all", "--verbose"]
status.code = 0
