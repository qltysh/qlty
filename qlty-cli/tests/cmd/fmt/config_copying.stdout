 JOBS: 1 

Plugin  Result   Targets   Time   Debug File
sample  Success  1 target  [..]s  .qlty/out/invoke-[..].yaml

Formatted 1 file
